Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -16,8 +19,11 @@
   name: uplink-revolver-python-name
   description: uplink-revolver-python-desc
   productEntity: WeaponRevolverPythonAP
-  cost:
-    Telecrystal: 8 # Originally was 13 TC but was not used due to high cost
+  discountCategory: usualDiscounts
+  discountDownTo:
+    Telecrystal: 4
+  originalCost:
```

Comment:
new name looks bad, it will break fork stuff, need to think about a way to prevent all of that